server/routes/api.js

```diff
@@ -329,6 +329,298 @@
     }
 
     /**
+     *
+     * @name getDogRatings
+     * @param {Object} request
+     * @param {Object} response
+     */
+    function getDogRatings(request, response) {
+        Dog.findOne({
+            _id: request.params.dogId
+        }).exec(function (error, dog) {
+            if (error) {
+                return errorResponse(response, error);
+            }
+            response.json({
+                count: dog.ratings.length || 0,
+                value: dog.ratings
+            });
+        });
+    }
+
+    /**
+     *
+     * @name getDogRating
+     * @param {Object} request
+     * @param {String} request.param.dogId
+     * @param {String} request.param.dogRatingId
+     * @param {Object} response
+     */
+    function getDogRating(request, response) {
+        var lookup;
```

Comment:
Just being picky: `var lookup` could be placed inside the `.exec(` function below, to reflect its true scope.